### ISSUE TITLE
VS code keybinding: move item up and down

### ISSUE THREAD

**Issue by OnigiriJack** [zed-industries/zed#4652]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

In vs code to move a line up it should be option key + arrow key up or down. Right now it seems to be using control key + command + arrow key up or down.

### Environment

Zed: v0.91.3 (stable)
OS: macOS 13.4.1
Memory: 32 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by gbrrrl-no** [comment]
I still have this issue!

**Comment by gbrrrl-no** [comment]
I think this should work as default.

**Comment by texastoland** [comment]
It looks like `alt-up` and `alt-down` are taken by `editor::SelectLargerSyntaxNode` and `editor::SelectSmallerSyntaxNode`. But I think those are from JetBrains. The VS Code defaults should be:

```json
{
    "context": "Editor",
    "bindings": {
      "cmd-shift-k": "editor::DeleteLine",
      "alt-shift-down": "editor::DuplicateLine",
      "alt-down": "editor::MoveLineDown",
      "alt-up": "editor::MoveLineUp",
      "ctrl-shift-left": "editor::SelectSmallerSyntaxNode",
      "ctrl-shift-right": "editor::SelectLargerSyntaxNode"
    }
  }
}
```

Should we just PR that?

**Comment by Moshyfawn** [comment]
I'm going to close this issue as the patch has already been shipped in [`v0.126.0`](https://zed.dev/releases/preview/0.126.0).

**Comment by texastoland** [comment]
> the patch has already been shipped in [`v0.126.0`](https://zed.dev/releases/preview/0.126.0).

Edit: 8352f39 now in [`v0.127.0-pre`](https://github.com/zed-industries/zed/releases/tag/v0.127.0-pre) (mentioned in Breaking changes) 🚀
